.github/workflows/pull request.yml

```diff
@@ -10,28 +10,22 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ windows-latest, ubuntu-latest, macos-13 ]
+        os: [ windows-latest, ubuntu-latest, macos-13, macos-14 ]
         framework_version: [ 'net9.0', 'net48' ]
         exclude:
-          - os: ubuntu-latest
-            framework_version: 'net48'
```

Comment:
Revert also these changes to prevent trying to test on .NET Framework 4.8 on `ubuntu-latest`.

Reply:
Should be sorted now.